### PR TITLE
Feature branch shivani

### PR DESCRIPTION
**Pull request by shivaniS707**

### CODE REVIEW

File: team-omega-ui-project/public/index.html [hunk 1]
```diff
@@ -1,32 +1,80 @@
...
+        </li>
+        <li class="nav-item ">
+          <a class="nav-link " href="#">TEAM CONTACTS </a>
+        </li>
+        <li class="nav-item ">
+          <a class="nav-link active" href="#">STORY BOOK <span class="sr-only">(current)</span></a>
+        </li>
+      </ul>
+    </div>
+  </nav> -->
```
Comment (Bhagavatteja123) [inline team-omega-ui-project/public/index.html:72]: Why do we need this ? when we are using layout-header component?

File: team-omega-ui-project/public/index.html [hunk 2]
```diff
@@ -1,32 +1,80 @@
...
+
+  <!-- <div class="row">
+    <div class="col-sm-4 col-md-2 col-lg-2 main-content1"> -->
+
+
```
Comment (Bhagavatteja123) [inline team-omega-ui-project/public/index.html:77]: Remove commented lines.

File: team-omega-ui-project/src/components/App.js [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
+import Layoutfooter from "./common/layout-footer";
+
+import content from "./Mock";
 class App extends Component {
   render() {
+   
+    // const routePath = content.map(({ component, dlink }, id) => 
+    // <Route key={id} path={dlink} component={component} />
+  // );
+
```
Comment (Bhagavatteja123) [inline team-omega-ui-project/src/components/App.js:19]: Remove commented lines.

**Review by Bhagavatteja123 — COMMENTED** [review]
LGTM. 
Once following review comments are addressed we can merge the PR.

**Comment by shivaniS707** [comment]
Hi Teja

I will look into it.

Regards
Shivani singh

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Bhagavat Teja ***@***.***>
Sent: Wednesday, July 21, 2021 3:49:22 PM
To: Bhagavatteja123/Omega-UI ***@***.***>
Cc: SHIVANI KACHHWAH ***@***.***>; Author ***@***.***>
Subject: [External] Newsletter/Marketing: Re: [Bhagavatteja123/Omega-UI] Feature branch shivani (#3)


[Warning !] Exercise caution – External Mail.

@Bhagavatteja123 commented on this pull request.

LGTM.
Once following review comments are addressed we can merge the PR.

________________________________

In team-omega-ui-project/public/index.html<https://clicktime.symantec.com/3E1v3J5bKswo3xXbM3pSzxA7Vc?u=https%3A%2F%2Fgithub.com%2FBhagavatteja123%2FOmega-UI%2Fpull%2F3%23discussion_r673846594>:

> +  


Why do we need this ? when we are using layout-header component?

________________________________

In team-omega-ui-project/public/index.html<https://clicktime.symantec.com/3XxSXyuJhrFNuELvri8bFBw7Vc?u=https%3A%2F%2Fgithub.com%2FBhagavatteja123%2FOmega-UI%2Fpull%2F3%23discussion_r673846912>:

> +
+  
+
+


Remove commented lines.

________________________________

In team-omega-ui-project/src/components/App.js<https://clicktime.symantec.com/3PCb5XbVTMcbFjUVXDPfigf7Vc?u=https%3A%2F%2Fgithub.com%2FBhagavatteja123%2FOmega-UI%2Fpull%2F3%23discussion_r673847118>:

> +
+    // const routePath = content.map(({ component, dlink }, id) =>
+    // <Route key={id} path={dlink} component={component} />
+  // );
+


Remove commented lines.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://clicktime.symantec.com/3Rv2CWe5sfsefVqSPkTqJ4z7Vc?u=https%3A%2F%2Fgithub.com%2FBhagavatteja123%2FOmega-UI%2Fpull%2F3%23pullrequestreview-711491070>, or unsubscribe<https://clicktime.symantec.com/3UcDoJxpe44mPiAHz6FexCY7Vc?u=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAUANLRVTZ27XWNVCBNNH5OLTY2NKVANCNFSM472LCADQ>.

Information transmitted by this e-mail is proprietary to Mphasis, its associated companies and/ or its customers and is intended for use only by the individual or entity to which it is addressed, and may contain information that is privileged, confidential or exempt from disclosure under applicable law. If you are not the intended recipient or it appears that this mail has been forwarded to you without proper authority, you are notified that any use or dissemination of this information in any manner is strictly prohibited. In such cases, please notify us immediately at ***@***.*** and delete this mail from your records.

**Review by Bhagavatteja123 — APPROVED** [review]
LGTM,

**Pull request merged**